### PR TITLE
ENH: PEP8 no changes, only YAPF on the code

### PR DESCRIPTION
**Pull request by MadsJensen**
Just cleaned up the code a bit. No changes only visual.

### CODE REVIEW

**Comment by MadsJensen** [comment]
Maybe do all the code before merge?

File: stormdb/cluster.py [hunk 1]
```diff
@@ -130,9 +128,15 @@ def get_load_dict(self):
...
-                            total=qq[5])]
-        return(q_list)
+            q_list += [
+                dict(
+                    name=qq[0],
+                    load=qq[1],
+                    used=qq[2],
+                    avail=qq[4],
+                    total=qq[5])
+            ]
```
Comment (cjayb) [inline stormdb/cluster.py:138]: ugh, really?

**Review by cjayb — APPROVED** [review]
Now I have flake8 up and running again in Atom, don't see any warnings so happy enough. There are some style issues though (hanging indents etc.), and I guess opening out the long argument lists is useful (though long-winded).

TL;DR I'm OK with running YAPF on all of the code, don't feel too strongly about it really. Will you do it? Merge if happy with the result.

File: stormdb/cluster.py (hunk 1 above)
Comment (MadsJensen) [reply stormdb/cluster.py:138]: yep ;), we can skip it though

**Comment by MadsJensen** [comment]
Added the access and base

**Comment by cjayb** [comment]
What the heck, figured we'd try this... Not sure I can stick to the formatting guidelines though!
